index.html

```diff
@@ -364,7 +364,7 @@
                     <table class="simple">
                         <thead><tr><th>Current state of device buttons</th><th><code>buttons</code></th></tr></thead>
                         <tbody>
-                            <tr><td><b>Mouse move with no buttons pressed</b></td><td>0</td></tr>
+                            <tr><td><b>Mouse moved with no buttons pressed</b><br>, Pen moved without changing its <a>active buttons state</a></td><td>0</td></tr>
```

Comment:
A quick thought: this seems more correct to replace the mouse/pen specific lines with:
"No change in active buttons state   =>   0"